go/libkb/skb.go

```diff
@@ -770,14 +766,14 @@ func (s *SKB) unlockPrompt(arg SecretKeyPromptArg, which string, secretStore Sec
 }
 
 // func (s *SKB) PromptAndUnlock(lctx LoginContext, reason, which string, secretStore SecretStore, ui SecretUI, lksPreload *LKSec, me *User) (ret GenericKey, err error) {
-func (s *SKB) PromptAndUnlock(arg SecretKeyPromptArg, which string, secretStore SecretStore, lksPreload *LKSec, me *User) (ret GenericKey, err error) {
+func (s *SKB) PromptAndUnlock(arg SecretKeyPromptArg, which string, username NormalizedUsername, lksPreload *LKSec, me *User) (ret GenericKey, err error) {
```

Comment:
since you're already passing in `me`, you don't need to pass in `username` separately